tools/codegen/model.py

```diff
@@ -1496,3 +1503,40 @@ def parse_returns(return_decl: str) -> Tuple[Return, ...]:
     if return_decl[0] == '(' and return_decl[-1] == ')':
         return_decl = return_decl[1:-1]
     return tuple(Return.parse(arg) for arg in return_decl.split(', '))
+
+@dataclass(frozen=True)
+class PrecomputedElement:
+    name: str
+    ty: str
+
+    @staticmethod
+    def build(src: str) -> 'PrecomputedElement':
```

Comment:
BTW, the preexisting nomenclature in the file is "parse" as opposed to "build" (since it's parsing a string)